feature_engine/selection/base_recursive_selector.py

```diff
@@ -132,7 +132,7 @@ def fit(self, X: pd.DataFrame, y: pd.Series):
         # self._confirm_variables(X)
 
         # find numerical variables or check variables entered by user
-        self.variables_ = _find_or_check_numerical_variables(X, self.variables_)
+        self.variables_ = _find_or_check_numerical_variables(X, self.variables)
```

Comment:
@solegalli This one was causing 11 tests to fail. Still 7 failed but we are going in the right direction!